src/pages/ErrorPage/UpdateRequiredView.tsx

```diff
@@ -37,17 +40,21 @@ function UpdateRequiredView() {
                 <View style={[styles.ph5, styles.alignItemsCenter, styles.mt5]}>
                     <View style={styles.updateRequiredViewTextContainer}>
                         <View style={[styles.mb3]}>
-                            <Text style={[styles.newKansasLarge, styles.textAlignCenter]}>{translate('updateRequiredView.pleaseInstall')}</Text>
+                            <Text style={[styles.newKansasLarge, styles.textAlignCenter]}>
+                                {isProduction ? translate('updateRequiredView.pleaseInstallExpensifyClassic') : translate('updateRequiredView.pleaseInstall')}
+                            </Text>
```

Comment:
I don't think this logic works across standalone AND Hybrid. HybridApp user would also see the `pleaseInstallExpensifyClassic` message... 

Can we lock the message behind an isStandalone condition?

Reply:
Right! fixed